.travis.yml

```diff
@@ -20,12 +20,18 @@ matrix:
       - cargo clippy --examples -- -D warnings
       - cargo doc --no-deps
       - make -C examples
+      - cd tools/httpbintest
```

Comment:
Pro tip: you can pass `--manifest-path` to `cargo build` and `cargo test` directly, so you don't need to cd into the directory.

Reply:
done, but I backed-out the change to run tests in Travis to avoid build failure noise